.eslintrc.json

```diff
@@ -1,5 +1,5 @@
 {
-  "extends": "./node_modules/aurelia-tools/.eslintrc.json",
+  "extends": "./../../aurelia-tools/.eslintrc.json",
```

Comment:
Why did this need to be changed? It worked fine for me and the path seems correct. The tools should be installed. Maybe the package.json was updated and you didn't install new deps?